### PR TITLE
Support for niimg in find_cut_slices

### PR DESCRIPTION
**Pull request by akshayah3**
#1193

### CODE REVIEW

**Comment by akshayah3** [comment]
I tried to add support for niimg in `find_cut_slices`. I will make further changes and add test cases based on the reviews.

File: nilearn/plotting/find_cuts.py [hunk 1]
```diff
@@ -178,7 +178,8 @@ def find_cut_slices(img, direction='z', n_cuts=7, spacing='auto'):
 
     Parameters
     ----------
-    img: 3D Nifti1Image
+    img: Niimg-like object
```
Comment (AlexandreAbraham) [inline nilearn/plotting/find_cuts.py]: Please specify 3D here.

Comment (akshayah3) [inline nilearn/plotting/find_cuts.py]: Made the change.

**Comment by AlexandreAbraham** [comment]
Thanks, will merge when green.

**Pull request merged**